Makefile

```diff
@@ -74,4 +74,8 @@ test-with-cov:
 	poetry run coverage erase
 	poetry run pytest --cov splunk_otel --cov-append --cov-branch --cov-report='' --junit-xml=test_results/results.xml tests/unit/
 	poetry run coverage report --show-missing
-	poetry run coverage xml
\ No newline at end of file
+	poetry run coverage xml
+
+.PHONY: create-github-release
+ci-create-github-release:
+	poetry run python scripts/create_gh_release.py --dry-run=false
```

Comment:
nit: I feel safer one there is a blank line at the end of files (especially the *nix files)

Reply:
Done. BTW do modern tools still not work with files that don't end with empty line well enough?